gridded/pyugrid/ugrid.py

```diff
@@ -844,7 +844,37 @@ def build_face_edge_connectivity(self):
         Not implemented yet.
 
         """
-        raise NotImplementedError
+        import pandas as pd
```

Comment:
as this is an optional import, let's wrap this with something like:

```
try:
    import pandas
except ImportError as err:
    err.args = ("The pandas package is required to compute the face_edge_connectivity",)
    raise
```

Reply:
alright, fine with me.

```suggestion
        try:
            import pandas
        except ImportError as err:
            err.args = ("The pandas package is required to compute the face_edge_connectivity",)
            raise
```